agent/instrumentation/micrometer-1.0/src/main/java/io/opentelemetry/javaagent/instrumentation/micrometer/ai/ActuatorInstrumentation.java

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package io.opentelemetry.javaagent.instrumentation.micrometer;
+package io.opentelemetry.javaagent.instrumentation.micrometer.ai;
```

Comment:
upstream adds a new package for app insights specifically?

Reply:
no, this is to avoid conflicting with upstream's package name (which already uses `io.opentelemetry.javaagent.instrumentation.micrometer`)